### ISSUE TITLE
NTR: CT or MRI: mass lesion in cerebellum or fourth ventricle (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8897]
New term request CT or MRI: mass lesion in cerebellum or fourth ventricle (norden_091220191110):   UMLS: 0742036

**Comment by pnrobinson** [comment]
@MickeySegal
what is the context? This is a bundled term and we would want to be more specific if possible

**Comment by MickeySegal** [comment]
It is better to use the existing bundled term "posterior fossa", so this should be "posterior fossa mass lesion", under https://hpo.jax.org/browse/term/HP:0000932

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001085
- label: Posterior fossa mass lesion
- parents: [Abnormal posterior cranial fossa morphology (HP:0000932)](https://hpo.jax.org/app/browse/term/HP:0000932)
- pmid: 
- def: An abnormal growth or accumulation of tissue in the posterior fossa, affecting the cerebellum or fourth ventricle. Typically a mass is an initial finding that will lead to additional workup.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8897
- synonyms:
